### PR TITLE
test wallet function change

### PR DESCRIPTION
**Pull request by winterweixiaodong**

### CODE REVIEW

File: logic/logic.go [hunk 1]
```diff
@@ -51,29 +53,19 @@ func CreateBlockchain(address core.Address, db storage.Storage, consensus core.C
...
-//create a wallet
-func CreateTestWallet() (*client.Wallet, error) {
-	path := strings.Replace(client.GetWalletFilePath(),"wallets","wallets_test",-1)
-	fl := storage.NewFileLoader(path)
-	wm := client.NewWalletManager(fl)
-	passBytes, err := bcrypt.GenerateFromPassword([]byte("test"), bcrypt.DefaultCost)
-	if err != nil {
-		return nil, err
+//create a wallet from path
+func CreateWalletWithPathPassword(path string, password string) (*client.Wallet, error) {
```
Comment (yeeshi) [inline logic/logic.go]: I think the name can be as short as "CreateWallet" since no duplicate functions

File: logic/logic.go [hunk 2]
```diff
@@ -205,3 +197,11 @@ func AddBalance(address core.Address, amount *common.Amount, bc *core.Blockchain
 	return err
 
 }
+
+func GetClientWalletFilePath() string {
+	return client.GetWalletFilePath()
+}
+
+func GetClientTestWalletFilePath() string {
```
Comment (yeeshi) [inline logic/logic.go]: The testing purpose function should be moved to testing files/packages

File: logic/logic.go (hunk 1 above)
Comment (winterweixiaodong) [reply logic/logic.go]: change the name to CreateWallet

File: logic/logic.go (hunk 2 above)
Comment (winterweixiaodong) [reply logic/logic.go]: removed

**Pull request merged**